learn/wavemaker-release-notes/v10-6-5.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "WaveMaker 10.6.5 - Release date: 25 February 2021"
+id: ""
+sidebar_label: "Release v10.6.5"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Bug Fixes
+
+---
+
+- Fixed an issue related to auto-generated tooltip in the Form and Field widgets.
```

Comment:
It is better to explain the fix, if you can @swethakundaram

Reply:
@prasanth-reddy-wavemaker-com, updated. Please check now.